app/overlays/mtl/dts_overlay.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_COMP_IIR=y
```

Comment:
I noticed this may be followed from:
./src/arch/xtensa/configs/override/mt8195_chrome_dts.config:4:CONFIG_DTS_CODEC=y
./src/arch/xtensa/configs/override/renoir_chrome_dts.config:3:CONFIG_DTS_CODEC=y
./src/arch/xtensa/configs/override/mt8186_chrome_dts.config:3:CONFIG_DTS_CODEC=y

my question is: is IIR must follow with DTS codec? they should have no relation from module perspecitve.

Reply:
We use EQ IIR + DTS to support customer project. For different platforms, EQ IIR might be default enabled or disabled. So we add it to make sure IIR is enabled. On MTL, IIR is default enabled. I could remove it if you think it's better.

Reply:
So the Kconfig can set a dependecy, we could use ```depends``` in Kconfig to state this relationship between DTS and IIR. 
@singalsu fyi.